pwndbg/heap/ptmalloc.py

```diff
@@ -177,22 +195,61 @@ def bk_nextsize(self):
 
         return self._bk_nextsize
 
-    # TODO Other useful methods e.g. next_chunk(), __iter__, __str__
+    @property
+    def arena(self):
+        if self._arena is None:
+            try:
+                ar_ptr = pwndbg.heap.current.get_heap(self.address)["ar_ptr"]
+                ar_ptr.fetch_lazy()
+            except Exception:
+                ar_ptr = None
+            if ar_ptr is not None and ar_ptr in (ar.address for ar in pwndbg.heap.current.arenas):
```

Comment:
I guess unless we expect `None` to appear as `ar.address`, we can skip the `ar_ptr is not None` check. But it's fine if you want to keep it for clarity.